locales/es/translation.json

```
@@ -160,7 +160,7 @@
         "current-month-title": "Ir al mes actual",
         "previous-month": "Mes anterior",
         "next-month": "Mes siguiente",
-        "time-to-leave": "Time to Leave",
+        "time-to-leave": "Time to leave",
```

Just missed capitalization of Leave:

```suggestion
        "time-to-leave": "Time to Leave",
```